mappings/net/minecraft/block/AbstractCandleBlock.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/class_5540 net/minecraft/block/AbstractCandleBlock
+	FIELD field_27083 LIT Lnet/minecraft/class_2746;
+	METHOD method_31610 spawnCandleParticles (Lnet/minecraft/class_1937;Lnet/minecraft/class_243;Ljava/util/Random;)V
+		ARG 0 world
+		ARG 1 vec3d
```

Comment:
```suggestion
		ARG 1 offset
```

Reply:
That is the actual position of the particle, and is calculated by `offset.add(pos.getX(), pos.getY(), pos.getZ())`, where `offset` is an offset `Vec3d` provided by `getParticleOffsets `/`method_31613` and `pos` is the position of a candle's display tick.